subprojects/build-init/src/main/resources/org/gradle/buildinit/tasks/templates/library-versions.properties

```diff
@@ -6,7 +6,7 @@ testng=6.10
 slf4j=1.7.22
 guava=21.0
 scala=2.11
-groovy=2.4.8
+groovy=2.4.9
```

Comment:
Changes to `library-versions.properties` are automated by our promotions jobs and should not be edited directly.
Could you please change it back to 2.4.8 for now?